packages/coil-extension/src/popup/components/views/TipConfirmView.tsx

```diff
@@ -104,26 +104,24 @@ export const TipConfirmView = (): React.ReactElement => {
   }
 
   const handleSubmit = async () => {
-    setFinalTipAmountUsd(currentTipAmountUsd)
-    router.to(ROUTES.tippingComplete)
-    // setSubmitError(null)
-    // setIsSubmitting(true)
-    // try {
-    //   // setting the final tip amount first
-    //   // this is done so that the tip context can properly update when storage is updated
-    //   setFinalTipAmountUsd(currentTipAmountUsd)
-    //   const { success, message } = await sendTip(currentTipAmountUsd)
+    setSubmitError(null)
```

Comment:
I had wondered about this code. 
Kinda the danger of leaving commented out blocks everywhere. 
It just becomes background noise.

Reply:
> There is no commented out code there?
There was until this commit :) 
For purposes of testing the anim without actually sending money I assume ?

Reply:
Yeah I merged that in a day ago, but you are correct it was for testing. I'm going to try reproducing your error again and if I can't I think i'm going to merge unless you have any other concerns.

Reply:
If you're happy with it. I'd probably put the animation wrappers in their own subfolder, and pull out a few consts at the least.

Reply:
If you're happy with it. I'd probably put the animation wrappers in their own subfolder, and pull out a few consts at the least.

Reply:
not sure what's up with the double comment there

Reply:
Agree with the organization of the wrappers. Moved them to their own directory and used a constant for the timing that is shared. Merging.